### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by apowers313** [Khan/KaTeX#739]
I'm trying to build KaTeX from scratch to see if the latest code has the issue I'm seeing in #634. I'm assuming that building KaTeX is as easy as typing `make` (I didn't see any build instructions).

When I try to build from the master branch on OSX, I get the following error:

> ./node_modules/.bin/eslint katex.js server.js cli.js src/Lexer.js src/MacroExpander.js src/Options.js src/ParseError.js src/Parser.js src/Settings.js src/Style.js src/buildCommon.js src/buildHTML.js src/buildMathML.js src/buildTree.js src/delimiter.js src/domTree.js src/environments.js src/fontMetrics.js src/fontMetricsData.js src/functions.js src/macros.js src/mathMLTree.js src/parseData.js src/parseTree.js src/stretchy.js src/symbols.js src/unicodeRegexes.js src/utils.js test/errors-spec.js test/katex-spec.js test/symgroups.js test/unicode-spec.js contrib/auto-render/auto-render-spec.js contrib/auto-render/auto-render.js contrib/auto-render/splitAtDelimiters.js dockers/Screenshotter/screenshotter.js dockers/texcmp/texcmp.js
/private/tmp/KaTeX/node_modules/browserify/bin/cmd.js -t [ babelify ] katex.js --standalone katex > build/katex.js
Error: Cannot find module 'babel-runtime/core-js/json/stringify' from '/private/tmp/KaTeX/src'
    at /private/tmp/KaTeX/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/private/tmp/KaTeX/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/private/tmp/KaTeX/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/private/tmp/KaTeX/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/private/tmp/KaTeX/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /private/tmp/KaTeX/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:123:15)
make: *** [build/katex.js] Error 1

Any thoughts on what might be causing this or how to go about debugging it?

**Comment by xymostech** [comment]
Hmm, interesting. It looks like maybe it failed to install some dependencies? If you run `npm install` and try again, does that help? What version of `npm` do you have?

I tried a fresh clone, and running `make` worked fine for me. I'm using node v7.10.0 and npm 4.2.0

**Comment by apowers313** [comment]
`npm install` hits the same error.

I'm on node v6.10.1 and npm 5.0.3 (just upgraded from 2.15.11).

**Comment by xymostech** [comment]
Interesting. If I use node v8.1.2 and npm 5.0.3, I also don't see this error. I also have a `node_modules/babel-runtime/core-js/json/stringify.js` file after running `npm install`. Do you have that file?

**Comment by edemaine** [comment]
FWIW, I have no trouble with Node v6.10.2 and npm 3.10.10.  Maybe there was a bad fetch -- delete the directory `node_modules` and try `npm install` again?  Alternatively, maybe we need to upgrade some dependency versions (e.g. babel)...

**Comment by apowers313** [comment]
I blew away the entire directory and did a fresh `git clone` and had the same problem.

Then I did `npm install -g babel-cli` and did a fresh `git clone` and ran `make` in (second) fresh clone and it worked.

To see if the global install of `babel-cli` was the issue, I did `npm uninstall -g babel-cli` and ran that in a (third) fresh clone, and it worked.

I'm not able to replicate the problem anymore, so maybe we close this out and re-open it if someone else has issues in the future.
